plugins/opencost/package.json

```diff
@@ -30,8 +30,8 @@
     "@material-ui/core": "^4.9.13",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.60",
-    "@material-ui/pickers": "^3.3.10",
     "@material-ui/styles": "^4.11.5",
+    "@mui/x-date-pickers": "6.17.0",
```

Comment:
sorry for the addon, but please commit the `yarn.lock` changes as well :)

Reply:
Will do this weekend